source/blog/2017-05-25-future-plans-for-fedora-atomic-release-life-cycle.html.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Future Plans for the Fedora Atomic Release Life Cycle
+author: jbrooks
+date: 2017-05-25 07:00:00 UTC
+tags: fedora
+comments: true
+published: true
+---
+
+The Fedora Project ships new releases at roughly six-month intervals, and [maintains](https://fedoraproject.org/wiki/Fedora_Release_Life_Cycle) each release for around thirteen months. So Fedora N is supported by the community until one month after the release of Fedora N+2.
+
+Since the first Fedora Atomic host shipped, as part of Fedora 21, the project has maintained separate ostree repositories for both of the active Fedora releases. For instance, there are currently trees available for Fedora Atomic 25 and Fedora Atomic 24.
+
+Fedora Atomic sets out to be a particularly fast-moving branch of Fedora, with releases every two weeks and updates to key “atomic” components such as docker and kubernetes that move more quickly than one might expect from Fedora.
+
+Due in part to this faster pace, the Fedora Atomic workgroup has always focused its testing and integration efforts most directly on the latest stable release, encouraging users of the older release to rebase to the newer tree as soon as possible. Releases older than the current tree are supported only on a "best effort" basis, meaning that the ostree is updated, but there is no organized testing of the older releases.
+
+Starting with either the Fedora 26 to 27 or the 27 to 28 upgrade cycle, the Fedora Atomic Workgroup intends to collapse Fedora Atomic into a single version which will track the latest stable Fedora branch. When a new stable version of Fedora is released, Fedora Atomic users will automatically shift to the new version when they install updates.
```

Comment:
again, getting caught up on technical details. From the VFAD we had a while back the exact [proposed way](https://pagure.io/atomic-wg/issue/231#comment-432157) to achieve a **latest** ref was `by having a stable or latest ref in our repo that is a symbolic link to the ref that follows a particular release.` Which by definition still means there are refs that follow particular releases.

I'm not encouraging people to stay on life supported releases, but they will have the option to follow a ref that is not `latest`. The most useful case for this in my mind is if I have my systems set up to automatically check for an update and update/reboot at midnight every night. For the 26->27 upgrade i'd rather not have that one be automated so I'll follow the 26 ref until I'm ready to do a manual upgrade to 27.